moped-editor/src/views/projects/projectView/ProjectStatusBadge.js

```diff
@@ -17,31 +17,27 @@ const defaultLabel = "Unknown";
 /**
  * Retrieves the style configuration for an individual phase
  * @param {Object} theme - The theme object
- * @param {string} phaseName - The phase name to be used as the badge label
- * @param {string} phaseKey - The key to be used to determine badge styles
+ * @param {string} phaseKey - The key used to determine badge styles
  * @returns {Object}
  */
 const getStyle = (theme, phaseKey) => {
   /**
    * Font colors
    */
   const primary = theme?.palette?.text?.primary;
-  const white = theme?.palette?.common?.white;
+  const white = theme?.palette?.background?.paper;
 
   /**
    * Background color mapping
    */
   const backgroundColors = {
     default: theme?.palette?.grey?.[300],
     warning: theme?.palette?.warning?.light,
-    success: theme?.palette?.success?.main,
-    info: theme?.palette?.info?.dark,
+    success: theme?.palette?.success?.light,
+    info: theme?.palette?.info?.main,
     error: theme?.palette?.error?.main,
   };
```

Comment:
as documented [here](https://mui.com/material-ui/migration/v5-style-changes/#change-default-theme-palette-warning-colors), the `warning` is darker:
- v4: `#ffb74d`
- v5: `#ED6C02`

should we listen to MUI wrt to accessibility and leave the darker color? i was thinking we may want to revisit the status badge pallet anyway as we turn our attention to the projects map.